intezer_sdk/results.py

```diff
@@ -0,0 +1,104 @@
+from typing import List, Dict, Tuple
```

Comment:
We always split one import in one line in the backend, you can config this in your formatter

Reply:
why?